latch_cli/utils.py

```diff
@@ -295,3 +306,14 @@ def __exit__(self, type, value, traceback):
 class WorkflowType(Enum):
     latchbiosdk = "latchbiosdk"
     snakemake = "snakemake"
+
+
+def identifier_suffix_from_str(x: str) -> str:
+    res = ""
+    for c in x:
+        res += c if f"_{c}".isidentifier() else "_"
+    return res
+
+
+def identifier_from_str(x: str) -> str:
+    return (x[0] if x[0].isidentifier() else "_") + identifier_suffix_from_str(x[1:])
```

Comment:
do you want to include x[0] in the identifier when `f"_{x[0]}"` is valid? as of now, `x[0]` can just disappear from the final identifier.